api/src/models/student.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require("mongoose");
+const uniqueValidator = require("mongoose-unique-validator");
+/**
+ * @swagger
+ *  components:
+ *   schemas:
+ *    Student:
+ *      type: object
+ *      required:
+ *        - name
+ *        - upi
+ *      properties:
+ *        name:
+ *          type: string
+ *          description: name of the student
+ *        studentID:
+ *          type: String
+ *          description: ID of the student
+ *        upi:
+ *          type: string
+ *          description: Students UPI
+ *        yearLevel:
+ *          type: integer
+ *          description: year level of the student
+ *        plans:
+ *          type: Object
+ *          description: the plans that the student has
+ */
+const studentSchema = mongoose.Schema({
+    name: {
+        type: String,
+        unique: false,
+        required: true,
+    },
+    studentID: {
+        type: String,
+        unique: true,
+        required: true,
```

Comment:
This isn't written as required in the swagger comment

Reply:
fixed